<!doctype html>
<html>
	<head>
		<meta charset="utf-8">
		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">

		<title>HBV Migration</title>
		<meta name="author" content="Barney Isaksen Potter">
		<meta name="description" content="Human migration patterns shape hepatitis B diversity" />

		<link rel="stylesheet" href="dist/reset.css">
		<link rel="stylesheet" href="dist/reveal.css">
		<link rel="stylesheet" href="dist/theme/white.css">
		<link rel="stylesheet" href="css/custom.css">
		<!-- Theme used for syntax highlighted code -->
		<link rel="stylesheet" href="plugin/highlight/monokai.css">
	</head>
	<body>
		<div class="reveal">
			<div class="slides">
				<!-- BEGIN SLIDES -->
				<!-- Title slide -->
				<section data-background-color="#ebdbb2" data-background-image="assets/share/sedes_t.png"
						 data-background-size="35%"
						 data-background-position="125% 125%"
						 data-background-repeat="no-repeat"
						 data-background-opacity="0.5">
					<h2 style="color:#076678">Human migration patterns shape hepatitis B diversity</h2>
					<h4 style="color:#076678">Barney Isaksen Potter</h4>
					<p style="font-size:0.6em;color:#282828">KU Leuven</p>
					<p style="font-size:0.6em;color:#282828">2022-12-21</p>
					<p style="font-size:0.6em;color:#282828">Life Sciences Center, Vilnius University</p>
					<aside class="notes">
						<ul>
							<li>Thanks for being here!</li>
							<li>I'm going to talk about how what Bayesian phylogeography is, and how we've applied it to an difficult dataset</li>
							<li>Help explain some of the history of HBV's introduction to Europe</li>
						</ul>
					</aside>
				</section>
				<section>
					<div class="r-stack">
						<img class="fragment current-visible" src="assets/share/phylogeo_intro/gruvbox_dark/epi1.png">
						<img class="fragment current-visible" src="assets/share/phylogeo_intro/gruvbox_dark/epi2.png">
						<img class="fragment current-visible" src="assets/share/phylogeo_intro/gruvbox_dark/epi3.png">
						<img class="fragment current-visible" src="assets/share/phylogeo_intro/gruvbox_dark/epi4.png">
						<img class="fragment current-visible" src="assets/share/phylogeo_intro/gruvbox_dark/epi5.png">
						<img class="fragment current-visible" src="assets/share/phylogeo_intro/gruvbox_dark/epi6.png">
						<img class="fragment current-visible" src="assets/share/phylogeo_intro/gruvbox_dark/epi7.png">
						<img class="fragment" src="assets/share/phylogeo_intro/gruvbox_dark/epi8.png">
					</div>
					<aside class="notes">
						<ul>
							<li>First: how epidemic process leads to phylogenetic trees</li>
							<li>walk through figure</li>
							<li>phylogeny represents a hypothesis of the shared evolutionary, and therefore transmission, history</li>
						</ul>
					</aside>
				</section>
				<!-- Phylogeny explanation slide -->
				<section data-transition="fade">
					<h3>What can phylogenetics tell us?</h3>
					<div id="left">
						<ul>
							<li>Phylogeny</li>
							<li style="opacity:.5">Evolutionary rate</li>
							<li style="opacity:.5">Substitution rates</li>
							<li style="opacity:.5">Demographic parameters</li>
							<li style="opacity:.5">Migration rates</li>
							<li style="opacity:.5">Migration history</li>
						</ul>
					</div>
					<div id="right">
						<img class="r-stretch" src="assets/share/what_can_phylogeo/gb_dark/phylogeny.png" width="100%">
					</div>
					<aside class="notes">
						<ul>
							<li>Which things are most related to one another</li>
							<li>What are the patterns of shared ancestry?</li>
							<li>How diverged two different taxa are (genetic distance)</li>
						</ul>
					</aside>
				</section>
				<!-- Evolutionary rate explanation slide -->
				<section data-transition="fade">
					<h3>What can phylogenetics tell us?</h3>
					<div id="left">
						<ul>
							<li style="opacity:.5">Phylogeny</li>
							<li>Evolutionary rate</li>
							<li style="opacity:.5">Substitution rates</li>
							<li style="opacity:.5">Demographic parameters</li>
							<li style="opacity:.5">Migration rates</li>
							<li style="opacity:.5">Migration history</li>
						</ul>
					</div>
					<div id="right">
						<img class="r-stretch" src="assets/share/what_can_phylogeo/gb_dark/time_scale.png" width="100%">
					</div>
					<aside class="notes">
						<ul>
							<li>From genetic distance to clock distance</li>
							<li>How long ago did two taxa share a common ancestor (and how long ago did they diverge)?</li>
							<li>Leverage clock-like nature of mutation accumulation to date internal nodes</li>
						</ul>
					</aside>
				</section>
				<!-- Substitution rate explanation slide -->
				<section data-transition="fade">
					<h3>What can phylogenetics tell us?</h3>
					<div id="left">
						<ul>
							<li style="opacity:.5">Phylogeny</li>
							<li style="opacity:.5">Evolutionary rate</li>
							<li>Substitution rates</li>
							<li style="opacity:.5">Demographic parameters</li>
							<li style="opacity:.5">Migration rates</li>
							<li style="opacity:.5">Migration history</li>
						</ul>
					</div>
					<div id="right">
						\[\tiny
						Q_{JC69} =
						\begin{bmatrix}
						\cdot & \lambda & \lambda & \lambda \\
						\lambda & \cdot & \lambda & \lambda \\
						\lambda & \lambda & \cdot & \lambda \\
						\lambda & \lambda & \lambda & \cdot \\
						\end{bmatrix} \\[1em]
						Q_{GTR} =
						\begin{bmatrix}
						\cdot & a\pi_C & b\pi_A & c\pi_G \\
						a\pi_T & \cdot & d\pi_A & e\pi_G \\
						b\pi_T & d\pi_C & \cdot & f\pi_G \\
						c\pi_T & e\pi_C & f\pi_A & \cdot \\
						\end{bmatrix}
						\]
					</div>
					<aside class="notes">
						<ul>
							<li>Nucleotide substitution rates</li>
							<li>Variety of models</li>
							<li>Form the backbone of phylogenetic inference because they inform the likelihood function</li>
						</ul>
					</aside>
				</section>
				<!-- Demographic parameters explanation slide -->
				<section data-transition="fade">
					<h3>What can phylogenetics tell us?</h3>
					<div id="left">
						<ul>
							<li style="opacity:.5">Phylogeny</li>
							<li style="opacity:.5">Evolutionary rate</li>
							<li style="opacity:.5">Substitution rates</li>
							<li>Demographic parameters</li>
							<li style="opacity:.5">Migration rates</li>
							<li style="opacity:.5">Migration history</li>
						</ul>
					</div>
					<div id="right">
						<img class="r-stretch" src="assets/share/what_can_phylogeo/gb_dark/demographic_history.png" width="100%">
					</div>
					<aside class="notes">
						<ul>
							<li>Not just evol. characteristics, but also demographic and epidemiological ones</li>
							<li>Use the phylogeny to infer effective population size through time</li>
							<li>proxy for reproductive rate</li>
						</ul>
					</aside>
				</section>
				<!-- Migration rate explanation slide -->
				<section data-transition="fade">
					<h3>What can phylogenetics tell us?</h3>
					<div id="left">
						<ul>
							<li style="opacity:.5">Phylogeny</li>
							<li style="opacity:.5">Evolutionary rate</li>
							<li style="opacity:.5">Substitution rates</li>
							<li style="opacity:.5">Demographic parameters</li>
							<li>Migration rates</li>
							<li style="opacity:.5">Migration history</li>
					</div>
					<div id="right">
						<img class="r-stretch" src="assets/share/what_can_phylogeo/gb_dark/migration_rates.png" width="100%">
					</div>
					<aside class="notes">
						<ul>
							<li>Similar to substitution rates, we can infer migration rates</li>
							<li>Here we treat location as discrete</li>
							<li>Help us understand which locations may be sources or sinks for viral spread</li>
						</ul>
					</aside>
				</section>
				<!-- Migration rate history slide -->
				<section data-transition="fade">
					<h3>What can phylogenetics tell us?</h3>
					<div id="left">
						<ul>
							<li style="opacity:.5">Phylogeny</li>
							<li style="opacity:.5">Evolutionary rate</li>
							<li style="opacity:.5">Substitution rates</li>
							<li style="opacity:.5">Demographic parameters</li>
							<li style="opacity:.5">Migration rates</li>
							<li>Migration history</li>
						</ul>
					</div>
					<div id="right">
						<img class="r-stretch" src="assets/share/what_can_phylogeo/gb_dark/migration_history.png" width="100%">
					</div>
					<aside class="notes">
						<ul>
							<li>Combine migration rates with a phylogenetic tree</li>
							<li>Understand the number and timing of different migration events</li>
							<ul>
								<li>Short term: understand the drivers of a local epidemic</li>
								<li>Long term: understand the complete evolutionary history of a virus in space and time</li>
							</ul>
						</ul>
					</aside>
				</section>
				<!-- METHODS -->
        <section>
          <blockquote>
            How can we infer phylogenies robustly from genomic sequence data and geographic metadata?
          </blockquote>
					<aside class="notes">
						How can we go about this? How can we...
					</aside>
        </section>
				<!-- Tree space size -->
				<section>
					<h3>Problem: parameter space size</h3>
					\[\tiny
					\begin{array}{cc}
					Num.~taxa & Num.~topologies \\ \hline
					1 & 1 \\
					2 & 1 \\
					3 & 3 \\
					4 & 15 \\
					5 & 105 \\
					6 & 945 \\
					7 & 10,395 \\
					8 & 135,135 \\
					9 & 2,027,025 \\
					\vdots & \vdots \\
					769 & 3.753 \times 10^{2,110}
					\end{array}
					\]
					<aside class="notes">
						<ul>
							<li>Computationally burdensome</li>
							<li>Super-exponential growth of labeled, rooted, bifurcating trees</li>
							<li>Context: 10^80 atoms in universe, 27x universe-in-atom</li>
							<li>This is only the tree topologies</li>
							<li>Takeaway: exhaustive search is impossible</li>
						</ul>
					</aside>
				</section>
				<section>
					<img class="r-stretch" src="assets/share/phylogeo_intro/stamatakis2020_tree_space.png" width="100%">
					<div id="leftSource">Source: Stamatakis & Kozlov (2020)</div>
					<aside class="notes">
						<ul>
							<li>We want to efficiently:</li>
							<ul>
								<li>find starting trees that explain the observed sequence data well</li>
								<li>throw out trees that don't</li>
								<li>refine those trees in a principled way</li>
							</ul>
							<li>For this, we need a good metric of "goodness" of phylogeny</li>
						</ul>
					</aside>
				</section>
				<section>
					<h3>Enter: Bayes' Theorem</h3>
					\[
					P(\theta|\textbf{D}) = \frac{P(\textbf{D} |\theta) \times P(\theta )}{P(\textbf{D})}
					\]
					<aside class="notes">
						<ul>
							<li>Thinking about this problem in a Bayesian manner affords us three main benefits:</li>
							<ul>
								<li>Gives us an objective way to compare two phylogenetic hypotheses (tree + parameters)</li>
								<li>Lets us understand distribution of posterior parameters (unlike ML)</li>
								<li>Allows us to inject prior knowledge about certain parameters</li>
							</ul>
							<li>Third is crucially important in highly dimensional models that may be pathological</li>
						</ul>
					</aside>
				</section>
				<!-- <section data-auto-animate>
					\[\footnotesize
					P(Tree, Parameters | \textbf{Data}) = \frac{P(\textbf{Data} | Tree, Parameters) \times P(Tree, Parameters)}{P(\textbf{Data})}
					\]
				</section> -->
				<section>
					<img class="r-stretch" src="assets/share/phylogeo_intro/beast_likelihood.png" alt="likelihood calculation" width="100%"/>
					<aside class="notes">
						<ul>
							<li>Expand and apply to our application</li>
							<li>Probability of a given ..... is proportional to the joint probability of .....</li>
						</ul>
					</aside>
				</section>
				<section>
					<img class="r-stretch" src="assets/share/phylogeo_intro/likelihood_surface.png" alt="likelihood calculation" width="100%"/>
					<aside class="notes">
						The result of this is a highly dimensional likelihood surface:
						<ul>
							<li>We want to characterize it</li>
						</ul>
					</aside>
				</section>
				<section>
					<img class="r-stretch" src="assets/hbv-vilnius-university/mcmc_illustration.png" alt="likelihood calculation" width="100%"/>
					<div id="leftSource">Source: Jin, Ju, & Jung (2019)</div>
					<aside class="notes">
						<ul>
							<li>We do this using MCMC</li>
							<ul>
								<li>Start with a random guess</li>
								<li>Change it a little</li>
								<li>If the new guess is better, accept</li>
								<li>If it is worse, accept proportional the likelihood ratio</li>
							</ul>
							<li>As we log the state of the chain over time we recover a distribution for each parameter</li>
						</ul>
					</aside>
				</section>
				<section data-background="https://revolution-computing.typepad.com/.a/6a010534b1db25970b019aff4a7bbc970d-pi" data-background-size="30em">
					<aside class="notes">
						<ul>
							<li>Visually, this is what the process looks like in 2 dimensions</li>
							<li>Note: by replicating this process multiple times, we can be more confident in results if parallel replicates converge to the same thing</li>
						</ul>
					</aside>
				</section>
				<section>
					<img class="r-stretch" src="assets/share/phylogeo_intro/beast_main.png" alt="likelihood calculation" width="70%"/>
					<aside class="notes">
						<ul>
							<li>All of the Bayesian phylogeography is performed in Beast 1.0</li>
							<li>The group that I'm in is primarily concerned with developing new functionality and implementing new models in BEAST</li>
						</ul>
					</aside>
				</section>
				<!-- HBV SPECIFIC STUFF -->
                <!-- HBV image -->
				<section>
					<img class="r-stretch" src="assets/hbv-vilnius-university/hbv-micrograph.jpg" alt="HBV micrograph" />
						<div id="leftSource">Source: CDC/ Dr. Erskine Palmer</div>
						<aside class="notes">
						<ul>
							<li>Change gears: main antagonist: HBV</li>
						</ul>
					</aside>
				</section>
				<section>
					<h3>What is HBV?</h3>
					<div id="left" style="font-size:82%">
						<ul>
							<li>Causative agent of hepatitis B</li>
							<ul>
								<li>Cirrhosis</li>
								<li>Hepatocellular carcinoma</li>
							</ul>
							<li>Spread through blood and bodily fluids</li>
							<li>Affects $\geq 350,000,000$ people annually</li>
							<li>Partially dsDNA genome</li>
							<ul>
								<li>Long strand: 3 kb</li>
								<li>Short strand: 2.2 kb</li>
							</ul>
						</ul>
					</div>
					<div id="right">
						<img class="r-stretch" src="assets/hbv-vilnius-university/hbv_genome_schematic.png" width="100%" alt="HBV genome" />
						<div id="rightSource">Source: Wikimedia Commons</div>
					</div>
					<aside class="notes">
						<ul>
							<li>Orthohepadnavirus</li>
							<li>reverse-transcribing partially dsDNA genome</li>
							<li>circular (-) sense strand, partial ()+) sense strand</li>
							<li>overlapping reading frames</li>
							<li>highly compact genome, with little room for mutation</li>
						</ul>
					</aside>
				</section>
				<!-- EID slide -->
				<section>
					<img class="r-stretch" src="assets/second-year-seminar/emerging-infectious-diseases-morens-fauci-2020.jpg" alt="EIDs" />
						<div id="leftSource">Source: Morens & Fauci (2020)</div>
						<aside class="notes">
						<ul>
							<li>HBV stands in interesting contrast to most viral pathogens, because it is not recently emerged</li>
							<li>Diseases such as West African Ebola, Zika in the Americias, SARS-CoV-2 have all emerged very recently</li>
							<li>Even the influenza viruses that affect global health today have emerged in the last 100 years</li>
						</ul>
					</aside>
				</section>
                <!-- HBV tree -->
				<section>
					<img class="r-stretch" src="assets/hbv-vilnius-university/full-hbv-tree.jpg" alt="full hbv tree" />
					<div id="leftSource">Source: Kocher et al. (2021)</div>
						<aside class="notes">
						<ul>
							<li>HBV has coevolved with humans for thousands of years</li>
              <li>Categorized into genotypes; focus:</li>
							<li>HBV A, D, E</li>
						</ul>
					</aside>
				</section>
				<section>
					<img class="r-stretch" src="assets/hbv-vilnius-university/hbv-dist.jpeg" width="70%" alt="hbv distribution">
					<div id="leftSource">Source: Shi et al. (2013)</div>
					<aside class="notes">
						<ul>
							<li>Strong ethnospatial patterns</li>
							<ul>
								<li>A: Europe, 6 subgenotypes</li>
								<li>D: cosmopolitan, 7 subgenotypes</li>
								<li>E: Sub-Saharan African, no subgenotype defs.</li>
							</ul>
							<li>Genotypes display different disease severity and response to treatment</li>
						</ul>
					</aside>
				</section>
				<section>
					<h3>Our study</h3>
					<ul>
						<li>133 HBV positive individuals who have recently immigrated from sub-Saharan Africa</li>
						<li>118 full genome sequences:</li>
						<ul>
							<li>47 HBV-A;</li>
							<li>7 HBV-D;</li>
							<li>64 HBV-E.</li>
						</ul>
					<li>Supplemented with all available high quality GenBank HBV genomes</li>
					</ul>
					<aside class="notes">
						<li>2005-2010</li>
						<li>583 HBV-A sequences (dated 1979–2014)</li>
						<li>764 HBV-D sequences (dated 1975–2012)</li>
						<li>234 HBV-E sequences (dated 1994–2010)</li>
					</aside>
				</section>
				<section>
					Reconstruct evolutionary and migration history of three HBV genotypes (A, D, E):
					<p></p>
					<ul>
						<li>Identify subgenotypes for novel sequences (A/D);</li>
						<li>Date the most recent common ancestor of each genotype;</li>
						<li>Characterize number and timing of major viral migrations.</li>
					</ul>
					<aside class="notes">
						Why?
						<ol>
							<li>Helps us characterize the diversity of incoming HBV to Belgium</li>
							<ul>
								<li>Useful for public health interventions</li>
							</ul>
							<li>Gives us a better understanding of the evol history of each genotype</li>
							<ul>
								<li>Also the evolutionary rate</li>
							</ul>
							<li>What has driven viral spread throughout the globe and into Europe?</li>
						</ol>
					</aside>
				</section>
				<section>
					<h4>Sampling bias complicates phyogeographic inference</h4>
					<img class="r-stretch" src="assets/hbv-vilnius-university/static_map_with_counts.png" alt="hbv sampling map" />
					<aside class="notes">
						<ul style="font-scale:50%">
							<li>Sampling bias can have a huge affect on our geographic inference</li>
							<li>54 countries; 165 Iran HBV-D, 13 singletons, 37 w/ < 5</li>
							<li>Splitting by continent we see some similar issues</li>
							<li>Regions were chose to make the most equitable split for all three genotypes</li>
							<li>Bonus: computational time is helped massively</li>
						</ul>
					</aside>
				</section>
				<section>
					<img class="r-stretch" src="assets/hbv-vilnius-university/muhlemann.png" alt="hbv sampling map" />
					<div id="leftSource">Source: Mülemann et al. (2018)</div>
					<aside class="notes">
						<li>Issue: clock rate of HBV is very hard to derive from empirical data because of its peculiar genome</li>
						<li>We seek to remedy by drawing on recent work on ancient HBV genomics to get more data and informative priors</li>
					</aside>
				</section>
				<section>
					\[\scriptsize
					\begin{array}{cccc}
					\textbf{Sequence} & \textbf{Genotype} & \textbf{Location} & \textbf{Age~(year)} \\ \hline
					\textrm{Rise386} & \textrm{HBV-A} & \textrm{Russia}^* & 4,114 \textrm{(2100 BCE)} \\
					\textrm{Rise387} & \textrm{HBV-A} & \textrm{Russia}^* & 4,278 \textrm{(2264 BCE)} \\
					\textrm{DA119} & \textrm{HBV-A} & \textrm{Slovakia} & 1,563 \textrm{(451 CE)} \\
					\textrm{DA195} & \textrm{HBV-A} & \textrm{Hungary} & 2,641 \textrm{(627 BCE)} \\ \hline
					\textrm{DA27} & \textrm{HBV-D} & \textrm{Kazakhstan} & 1,610 \textrm{(409 CE)} \\
					\textrm{DA29} & \textrm{HBV-D} & \textrm{Kazakhstan} & 822 \textrm{(1197 CE)} \\
					\textrm{DA51} & \textrm{HBV-D} & \textrm{Kyrgyzstan} & 2,297 \textrm{(278 BCE)} \\
					\textrm{DA222} & \textrm{HBV-D} & \textrm{Kazakhstan} & 1,167 \textrm{(852 CE)} \\
					\textrm{NASD24SEQ} & \textrm{HBV-D} & \textrm{Italy} & 427 \textrm{(1587 CE)}
					\end{array}
					\]

					<p style="font-size:50%">Clock rate prior: $1.18 \times 10^{-5}$ [95% HPD: $8.04 \times 10^{-6} \textrm{--} 1.51 \times 10^{-5}$] subs./site/year</p>

					<div id="leftSource">Source: Mülemann et al. (2018), Ross et al. (2018)</div>

					<aside class="notes">
						<ul>
							<li>searched literature for high quality full ancient genomes</li>
							<li>4 HBV-A, 5 HBV-D</li>
							<li>None for HBV-E</li>
							<li>*Russian sequences are given a Central Asia location instead of Europe; proximal to Kazakhstan</li>
						</ul>
					</aside>
				</section>
				<!-- HBV-A -->
				<section>
					<h2>HBV-A</h2>
					<aside class="notes">
						583 + 4 ancient
					</aside>
				</section>
				<section>
					<h4>ML sequence placement</h4>
					<img class="r-stretch" src="assets/hbv-vilnius-university/HBV-A_ML.png" width="100%">
					<aside class="notes">
						<ul>
							<li>Maximum likelihood midpoint rooted phylogeny colored by subgenotype</li>
							<li>47 new genomes</li>
							<li>29 subgenotype 1</li>
							<li>6 subgenotype 6</li>
							<li>Takeaway: high clustering in Subgenotype 1 could have a public health implication in BE</li>
						</ul>
					</aside>
				</section>
				<section>
					<h4>Bayesian phylogeography</h4>
					<img class="r-stretch" src="assets/hbv-vilnius-university/HBV-A_phylogeography_and_mcc_tree.png" width="100%">
					<aside class="notes">
						<ul>
							<li>Time calibrated MCC tree colored by location</li>
							<li>Broken axis, swoops on map</li>
							<li>2500 BCE tMRCA, introduction to Europe shortly after</li>
							<li>Potentially drivers of mid-millenium diversity?</li>
							<ul>
								<li>5 jumps Africa to Americas after 1700</li>
								<li>1520's-1850s</li>
							</ul>
							<li>Globalization after the 1960s</li>
						</ul>
					</aside>
				</section>
				<section>
					<h4>Markov jump analysis</h4>
					<div id="left">
						<img class="r-stretch" src="assets/hbv-vilnius-university/mj_hbva.png" width="100%" alt="hbv a markov jumps" />
					</div>
					<div id="right">
						<img class="r-stretch" src="assets/hbv-vilnius-university/bf_hbva.png" width="100%" alt="hbv a bayes factors" />
					</div>
					<aside class="notes">
						<ul>
							<li>Europe drives exports</li>
							<li>Mainly to Americas and East Asia</li>
							<li>>11 Markov jumps each (with BF support)</li>
						</ul>
					</aside>
				</section>
				<!-- HBV-D -->
				<section>
					<h2>HBV-D</h2>
					<aside class="notes">
						764 + 5 ancient
					</aside>
				</section>
				<section>
					<h3>HBV-D new sequences</h3>
					<img class="r-stretch" src="assets/hbv-vilnius-university/HBV-D_ML.png" width="100%">
					<aside class="notes">
						<ul>
							<li>Subgenotypes 1, 2, 7</li>
							<li>Conclusion: minimal evidence that HBV-D is a driver of new European HBV</li>
						</ul>
					</aside>
				</section>
				<section>
					<h3>HBV-D evolutionary history</h3>
					<img class="r-stretch" src="assets/hbv-vilnius-university/HBV-D_phylogeography_and_mcc_tree.png" width="100%">
					<aside class="notes">
						<ul>
							<li>1800 BCE tMRCA</li>
							<li>92 total jumps</li>
							<ul>
								<li>more distributed throughout the evol histoyr</li>
								<li>matches: no geographic isolation allows continuous movement</li>
							</ul>
							<li>Large introduction to Africa ca. 550 CE</li>
							<li>Mixed European/American clade w/ tMRCA 1100 CE</li>
						</ul>
					</aside>
				</section>
				<section>
					<h4>Markov jump analysis</h4>
					<div id="left">
						<img class="r-stretch" src="assets/hbv-vilnius-university/mj_hbvd.png" width="100%" alt="hbv d markov jumps" />
					</div>
					<div id="right">
						<img class="r-stretch" src="assets/hbv-vilnius-university/bf_hbvd.png" width="100%" alt="hbv d bayes factors" />
					</div>
					<aside class="notes">
						<ul>
							<li>Central Asia, and to a smaller degree Europe drove spread</li>
							<li>makes sense given its known distribution</li>
						</ul>
					</aside>
				</section>
				<!-- HBV-E -->
				<section>
					<h2>HBV-E</h2>
					<aside class="notes">
						234 (no ancient)
					</aside>
				</section>
				<section>
					<h3>HBV-E new sequences</h3>
					<img class="r-stretch" src="assets/hbv-vilnius-university/HBV-E_ML.png" width="100%">
					<aside class="notes">
						<ul>
							<li>New sequences span diversity of tree</li>
							<li>Takeaway: perhaps systematic sampling bias</li>
						</ul>
					</aside>
				</section>
				<section>
					<h3>HBV-E evolutionary history</h3>
					<img class="r-stretch" src="assets/hbv-vilnius-university/HBV-E_phylogeography_and_mcc_tree.png" width="100%">
					<aside class="notes">
						<ul>
							<li>tMRCA 1825</li>
							<li>Almost all the history is in Africa</li>
							<li>All exportations are singletons, so their timing holds a lot of uncertainty</li>
						</ul>
					</aside>
				</section>
				<section>
					<h4>Markov jump analysis</h4>
					<div id="left">
						<img class="r-stretch" src="assets/hbv-vilnius-university/mj_hbve.png" width="100%" alt="hbv e markov jumps" />
					</div>
					<div id="right">
						<img class="r-stretch" src="assets/hbv-vilnius-university/bf_hbve.png" width="100%" alt="hbv e bayes factors" />
					</div>
					<aside class="notes">
						Africa to Europe is the main movement pattern
					</aside>
				</section>
				<section>
					<h3>Takeaways and next steps</h3>
					<ul>
						<li>Large-scale human movements may drive HBV globalization.</li>
						<li>Phylogeographic inference may be refined through joint inference with phylogeny.</li>
						<li>More global genomic surveillance of HBV is necessary to understand the diversity and potential impact of each genotype.</li>
						<li>Incorporation of more ancient genomes as they become available may refine inferences of future HBV datasets.</li>
					</ul>
					<aside class="notes">
						just read this slide
					</aside>
				</section>
				<!-- Acknowledgements slide -->
				<section data-background="#ffffff">
					<div class="row">
					<div class="column">
						<h4>ECV-KU Leuven</h4>
						<ul style="color:#282828">
							<li>Guy Baele</li>
							<li>Mandev Gill</li>
							<li>Philippe Lemey</li>
							<li>Marijn Thijssen</li>
							<li>Samuel Hong</li>
						</ul>
						<p></p>
						<h4>Fogarty International Center</h4>
						<ul style="color:#282828">
							<li>Nídia Sequeira Trovão</li>
						</ul>
						<p></p>
						<h4>Vilnius University</h4>
						<ul style="color:#282828">
							<li>Gytis Dudas</li>
						</ul>
					</div>
					<div class="column">
						<img class="r-stretch" src="assets/share/kul_logo.png" alt="KU Leuven logo" width="60%"/>
						<img class="r-stretch" src="assets/share/ecv_logo.png" alt="ecv logo" width="40%"/>
						<img class="r-stretch" src="assets/share/fogarty_logo.png" alt="Fogarty logo" width="60%"/>
						<img class="r-stretch" src="assets/share/vilnius_university_logo.png" alt="baef logo" width="70%"/>
						<img class="r-stretch" src="assets/share/baef_logo.png" alt="baef logo" width="60%"/>
					</div>
					<div class="column">
						<ul style="color:#282828">
							<li>Andrea Pineda-Peña</li>
							<li>Marijke Reynders</li>
							<li>Thomas Mina</li>
							<li>Carolina Alvarez</li>
							<li>Samad Amini-Bavil-Olayaee</li>
							<li>Frederik Nevens</li>
							<li>Piet Maes</li>
							<li>Marc Van Ranst</li>
							<li>Mahmoud Reza Pourkarim</li>
						</ul>
						<p></p>
						<h4>Belgian American Education Foundation</h4>
					</div>
					<aside class="notes">
						<ul>
							<li>baef gave me money</li>
						</ul>
					</aside>
				</section>
			</div>
		</div>

		<script src="dist/reveal.js"></script>
		<script src="plugin/notes/notes.js"></script>
		<script src="plugin/markdown/markdown.js"></script>
		<script src="plugin/highlight/highlight.js"></script>
		<script src="plugin/math/math.js"></script>
		<script>
			// More info about initialization & config:
			// - https://revealjs.com/initialization/
			// - https://revealjs.com/config/
		 Reveal.initialize({
			    controls: true,
				hash: true,
				slideNumber: true,
				hashOneBasedIndex: true,

				// Learn about plugins: https://revealjs.com/plugins/
				plugins: [ RevealMarkdown, RevealHighlight, RevealNotes, RevealMath.KaTeX, ]
			});
		</script>
	</body>
</html>
