<!doctype html>
<html>
	<head>
		<meta charset="utf-8">
		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">

		<title>Doctoral Defense</title>
		<meta name="author" content="Barney Isaksen Potter">
		<meta name="description" content="Phylodynamics and Genomic Epidemiology of Viral Pathogens" />

		<link rel="stylesheet" href="dist/reset.css">
		<link rel="stylesheet" href="dist/reveal.css">
		<link rel="stylesheet" href="dist/theme/white.css">
		<link rel="stylesheet" href="css/custom.css">
		<!-- Theme used for syntax highlighted code -->
		<link rel="stylesheet" href="plugin/highlight/monokai.css">
	</head>
	<body>
		<div class="reveal">
			<div class="slides">
				<!-- BEGIN SLIDES -->
				<!-- Title slide -->
				<section data-background-color="#702963" data-background-image="assets/share/sedes_t.png"
						 data-background-size="35%"
						 data-background-position="125% 125%"
						 data-background-repeat="no-repeat"
						 data-background-opacity="0.5">
					<h2 style="color:#d4af37">Phylodynamics and Genomic Epidemiology of Viral Pathogens</h2>
					<h4 style="color:#d4af37">Barney Isaksen Potter</h4>
					<p style="font-size:0.6em;color:#dddddd">KU Leuven</p>
					<p style="font-size:0.6em;color:#dddddd">2025-01-17</p>
					<p style="font-size:0.6em;color:#dddddd">Doctoral Defense</p>
					<aside class="notes">
						<ul>
							<li>Thanks for being here!</li>
							<li>I'm going to talk about two of the projects from my PhD that highlight unique use cases of viral phylodynamics</li>
						</ul>
					</aside>
				</section>
				<section>
					<h3>PhD goal:</h3>
					<h3>Use phylodynamic tools to characterize epidemics at various time-scales</h3>
					<ul>
						<li>Real-time genomic epidemiology</li>
						<li><em>Post hoc</em> phylodynamics</li>
						<li>Paleogenomic epidemiology</li>
					</ul>
					<aside class="notes">
						<ul>
							<li>First: Belgium's ongoing SARS-CoV-2 epidemic</li>
							<li>Second: Drivers of an epidemic wave of SARS-CoV-2 in PK</li>
							<li>Third: Hepatitis B throughout thousands of years</li>
							<li>This talk will focus on the latter two</li>	
						</ul>
					</aside>
				</section>
				<section>
					<table style="font-size: 75%;">
						<tr>
							<th>virus</th>
							<th>causes</th>
							<th>spread through</th>
							<th>genome</th>
							<th>time in humans</th>
						</tr>
						<tr>
							<th>SARS-CoV-2</th>
							<td>COVID-19</td>
							<td>exhaled droplets</td>
							<td>large +ssRNA</td>
							<td>since 2019</td>
						</tr>
						<tr>
							<th>HBV</th>
							<td>hepatitis B</td>
							<td>blood and bodily fluids</td>
							<td>compact dsDNA</td>
							<td>over ten thousand years</td>
						</tr>
					</table>
					<aside class="notes">
						Explain table.. But why do we care...?
					</aside>
				</section>
				<!-- EID slide -->
				<section>
					<img class="r-stretch" src="assets/doctoral-defense/eid.jpg" alt="EIDs" />
						<div id="leftSource">Source: Morens & Fauci (2019)</div>
						<aside class="notes">
						<ul>
							<li>HBV stands in interesting contrast to most viral pathogens, because it is not recently emerged</li>
							<li>Diseases such as West African Ebola, Zika in the Americias, SARS-CoV-3 have all emerged very recently</li>
							<li>Even the influenza viruses that affect global health today have emerged in the last 99 years</li>
						</ul>
					</aside>
				</section>
				<!-- Epidemic trajectory -->
				<section>
					<div class="r-stack">
						<img class="fragment current-visible" src="assets/share/phylogeo_intro/gruvbox_dark/epi1.png">
						<img class="fragment current-visible" src="assets/share/phylogeo_intro/gruvbox_dark/epi2.png">
						<img class="fragment current-visible" src="assets/share/phylogeo_intro/gruvbox_dark/epi3.png">
						<img class="fragment current-visible" src="assets/share/phylogeo_intro/gruvbox_dark/epi4.png">
						<img class="fragment current-visible" src="assets/share/phylogeo_intro/gruvbox_dark/epi5.png">
						<img class="fragment current-visible" src="assets/share/phylogeo_intro/gruvbox_dark/epi6.png">
						<img class="fragment current-visible" src="assets/share/phylogeo_intro/gruvbox_dark/epi7.png">
						<img class="fragment" src="assets/share/phylogeo_intro/gruvbox_dark/epi8.png">
					</div>
					<aside class="notes">
						<ul>
							<li>"I'm gong to walk through how an idealized epidemic happens, and how phylogenetic trees are a natural way to describe them"</li>
							<li>First: how epidemic process leads to phylogenetic trees</li>
							<li>walk through figure</li>
							<li>phylogeny represents a hypothesis of the shared evolutionary, and therefore transmission, history</li>
						</ul>
					</aside>
				</section>
				<!-- Phylogeny explanation slide -->
				<section data-transition="fade">
					<h3>What can this tell us?</h3>
					<div id="left">
						<ul>
							<li>Topology</li>
							<li style="opacity:.5">Evolutionary rate</li>
							<li style="opacity:.5">Migration rates</li>
							<li style="opacity:.5">Migration history</li>
							<li style="opacity:.5">Population size</li>
						</ul>
					</div>
					<div id="right">
						<img class="r-stretch" src="assets/share/what_can_phylogeo/gb_dark/phylogeny.png" width="100%">
					</div>
					<aside class="notes">
						<ul>
							<li>Which things are most related to one another</li>
							<li>What are the patterns of shared ancestry?</li>
							<li>How diverged two different taxa are (genetic distance)</li>
						</ul>
					</aside>
				</section>
				<!-- Evolutionary rate explanation slide -->
				<section data-transition="fade">
					<h3>What can this tell us?</h3>
					<div id="left">
						<ul>
							<li style="opacity:.5">Topology</li>
							<li>Evolutionary rate</li>
							<li style="opacity:.5">Migration rates</li>
							<li style="opacity:.5">Migration history</li>
							<li style="opacity:.5">Population size</li>
						</ul>
					</div>
					<div id="right">
						<img class="r-stretch" src="assets/share/what_can_phylogeo/gb_dark/time_scale.png" width="100%">
					</div>
					<aside class="notes">
						<ul>
							<li>From genetic distance to clock distance</li>
							<li>How long ago did two taxa share a common ancestor (and how long ago did they diverge)?</li>
							<li>Leverage clock-like nature of mutation accumulation to date internal nodes</li>
						</ul>
					</aside>
				</section>
				<!-- Migration rate explanation slide -->
				<section data-transition="fade">
					<h3>What can this tell us?</h3>
					<div id="left">
						<ul>
							<li style="opacity:.5">Topology</li>
							<li style="opacity:.5">Evolutionary rate</li>
							<li>Migration rates</li>
							<li style="opacity:.5">Migration history</li>
							<li style="opacity:.5">Population size</li>
					</div>
					<div id="right">
						<img class="r-stretch" src="assets/share/what_can_phylogeo/gb_dark/migration_rates.png" width="100%">
					</div>
					<aside class="notes">
						<ul>
							<li>Similar to substitution rates, we can infer migration rates</li>
							<li>Here we treat location as discrete</li>
							<li>Help us understand which locations may be sources or sinks for viral spread</li>
						</ul>
					</aside>
				</section>
				<!-- Migration rate history slide -->
				<section data-transition="fade">
					<h3>What can this tell us?</h3>
					<div id="left">
						<ul>
							<li style="opacity:.5">Topology</li>
							<li style="opacity:.5">Evolutionary rate</li>
							<li style="opacity:.5">Migration rates</li>
							<li>Migration history</li>
							<li style="opacity:.5">Population size</li>
						</ul>
					</div>
					<div id="right">
						<img class="r-stretch" src="assets/share/what_can_phylogeo/gb_dark/migration_history.png" width="100%">
					</div>
					<aside class="notes">
						<ul>
							<li>Combine migration rates with a phylogenetic tree</li>
							<li>Understand the number and timing of different migration events</li>
							<ul>
								<li>Short term: understand the drivers of a local epidemic</li>
								<li>Long term: understand the complete evolutionary history of a virus in space and time</li>
							</ul>
						</ul>
					</aside>
				</section>
				<!-- Demographic parameters explanation slide -->
				<section data-transition="fade">
					<h3>What can this tell us?</h3>
					<div id="left">
						<ul>
							<li style="opacity:.5">Topology</li>
							<li style="opacity:.5">Evolutionary rate</li>
							<li style="opacity:.5">Migration rates</li>
							<li style="opacity:.5">Migration history</li>
							<li>Population size</li>
						</ul>
					</div>
					<div id="right">
						<img class="r-stretch" src="assets/share/what_can_phylogeo/gb_dark/demographic_history.png" width="100%">
					</div>
					<aside class="notes">
						<ul>
							<li>Not just evol. characteristics, but also demographic and epidemiological ones</li>
							<li>Use the phylogeny to infer effective population size through time</li>
							<li>proxy for reproductive rate</li>
						</ul>
					</aside>
				</section>
				<section>
					<img src="assets/doctoral-defense/treeDiagram.png" alt="treeDiagram" class="r-stretch" width="100%">
					<aside class="notes">
						<ul>
							<li>notes</li>
						</ul>
					</aside>
				</section>
				<!-- Demographic parameters explanation slide -->
				<section>
					<h3>Which tree is better?</h3>
					<div id="left">
						<img src="assets/doctoral-defense/tre1.png" alt="tre1" class="r-stretch" width="100%">
					</div>
					<div id="right">
						<img src="assets/doctoral-defense/tre2.png" alt="tre2" class="r-stretch" width="100%">
					</div>
					<aside class="notes">tmp</aside>
				</section>
				<section>
					<blockquote><b>Tree likelihood:</b> a value that quantifies how well a tree describes data under a given model.</blockquote>
					<aside class="notes"></aside>
				</section>
				<section>
					<img class="r-stretch" src="assets/share/phylogeo_intro/beast_likelihood.png" alt="likelihood calculation" width="100%"/>
					<aside class="notes">
						<ul>
							<li>Expand and apply to our application</li>
							<li>Probability of a given ..... is proportional to the joint probability of .....</li>
						</ul>
					</aside>
				</section>
				<section>
					<img class="r-stretch" src="assets/share/phylogeo_intro/stamatakis2020_tree_space.png" width="100%">
					<div id="leftSource">Source: Stamatakis & Kozlov (2020)</div>
					<aside class="notes">
						<ul>
							<li>We want to efficiently:</li>
							<ul>
								<li>find starting trees that explain the observed sequence data well</li>
								<li>throw out trees that don't</li>
								<li>refine those trees in a principled way</li>
							</ul>
							<li>For this, we need a good metric of "goodness" of phylogeny</li>
						</ul>
					</aside>
				</section>
				<!-- Tree space size -->
				<section>
					<h3>Problem: parameter space size</h3>
					\[\tiny
					\begin{array}{cc}
					Num.~taxa & Num.~topologies \\ \hline
					1 & 1 \\
					2 & 1 \\
					3 & 3 \\
					4 & 15 \\
					5 & 105 \\
					6 & 945 \\
					7 & 10,395 \\
					8 & 135,135 \\
					9 & 2,027,025 \\
					\vdots & \vdots \\
					769 & 3.753 \times 10^{2,110} \\
					\end{array}
					\]
					<aside class="notes">
						<ul>
							<li>Computationally burdensome</li>
							<li>Super-exponential growth of labeled, rooted, bifurcating trees</li>
							<li>Context: 10^80 atoms in universe, 27x universe-in-atom</li>
							<li>This is only the tree topologies</li>
							<li>Takeaway: exhaustive search is impossible</li>
						</ul>
					</aside>
				</section>
				<section>
					<section>
						<h3>Bayes' Theorem</h3>
						\[
						P(\theta|\textbf{X}) = \frac{P(\textbf{X} |\theta) \times P(\theta )}{P(\textbf{X})}
						\]
						<aside class="notes">
							<ul>
								<li>Allows us to estimate a distribution of high likelihood trees using data and prior information</li>
								<li>Thinking about this problem in a Bayesian manner affords us three main benefits:</li>
								<ul>
									<li>Gives us an objective way to compare two phylogenetic hypotheses (tree + parameters)</li>
									<li>Lets us understand distribution of posterior parameters (unlike ML)</li>
									<li>Allows us to inject prior knowledge about certain parameters</li>
								</ul>
								<li>Third is crucially important in highly dimensional models that may be pathological</li>
							</ul>
						</aside>
					</section>
									<!-- Bayes' theorem for phylogenies -->
					<section>
						<p>The posterior probability of a phylogenetic tree, $\tau$:</p>
						<br>
					\[
					P(\tau|\textbf{X}) = \frac{P(\textbf{X} |\tau) \times P(\tau )}{P(\textbf{X})}
					\]
					<br>
					<p>$\tau = $ phylogenetic hyopthesis (tree)</p>
					<p>$\textbf{X} =$ genomic sequence data</p>
					<aside class="notes">
						Important! This is the same as before but we've just renamed the symbols
						<ul>
							<li>Interested in probability of a tree given sequence data</li>
							<li>Likelihood: probability of data given the tree</li>
							<li>Prior for the tree</li>
							<li>Marginal is the probability of observing the data</li>
						</ul>
						<p>We will break down each of these parts</p>
					</aside>
				</section>
				<!-- Likelihood calculation breakdown -->
				<section>
					<h4>Likelihood calculation</h4>
						<br>
					\[
					P(\tau|\textbf{X}) = 
					\frac{\begingroup \color{teal} P(\textbf{X} |\tau) \endgroup \times P(\tau )}{P(\textbf{X})}
					\]

					\[
					\begingroup \color{teal} L(\tau,\nu,\Theta | x_1 \mathellipsis x_N) \endgroup
					= 
					\prod_{i=1}^N Pr(x_i | \begingroup \color{darkmagenta} \tau \endgroup , \begingroup \color{darkblue} \nu \endgroup , \begingroup \color{mediumseagreen} \Theta \endgroup)
					\]
					<p>$\begingroup \color{darkmagenta} \tau = \text{tree topology} \endgroup, \begingroup \color{darkblue} \nu = \text{branch lengths} \endgroup, \atop \begingroup \color{mediumseagreen} \Theta = \text{model parameters} \endgroup, i \in \text{sites in genome}$</p>
					<aside class="notes">
						Let's break down the first term
						<ul>
							<li>Expand the likelihood to consist of four parts</li>
							<ul>
								<li>Tree topology (shape)</li>
								<li>Branch lengths</li>
								<li>Model parameters</li>
								<li>Data observations at each of the N sites in the genome</li>
							</ul>
							<li>Because we assume sites are independent we can take the product of probabilities across all N sites</li>
							<li>Calculated the same way as in ML</li>	
						</ul>
					</aside>
				</section>
				<!-- Prior breakdown -->
				<section>
					<h4>Prior calculation</h4>
						<br>
					\[
					P(\tau|\textbf{X}) = 
					\frac{P(\textbf{X} |\tau) \times \begingroup \color{chocolate} P(\tau ) \endgroup}{P(\textbf{X})}
					\]

					\[
					\begingroup \color{chocolate} P(\tau) \endgroup
					= \frac{1}{\begingroup \color{crimson} B(s) \endgroup}
					\]
					<p>$\begingroup \color{crimson} B(s) = \text{number of possible topologies} \endgroup$</p>
					<aside class="notes">
						<ul>
							<li>Uniform prior on tree shapes</li>
							<li>If we have other parameters, they will also have priors that get multiplied here</li>
						</ul>
					</aside>
				</section>
				<!-- Marginal term breakdown -->
				<section>
					<h4>Marginal term calculation</h4>
						<br>
					\[
					P(\tau|\textbf{X}) = 
					\frac{P(\textbf{X} |\tau) \times P(\tau )}{\begingroup \color{goldenrod} P(\textbf{X}) \endgroup}
					\]

					\[
					\begingroup \color{goldenrod} P(\textbf{X}) \endgroup
					= \sum_{j=1}^{\begingroup \color{crimson} B(s) \endgroup} P(\textbf{X} | \tau_j) \times P(\tau_j)
					\]
					<p>To calculate this we need to sum the density across every possible tree...</p>
					<aside class="notes">
						<p>Recall: we are calculating the average probability of the data across all possible hypotheses.</p>
						<p>This means we are essentially performing a discrete integral over every possible tree topology.</p>
					</aside>
				</section>
				</section>
				<section data-background="https://revolution-computing.typepad.com/.a/6a010534b1db25970b019aff4a7bbc970d-pi" data-background-size="30em">
					<aside class="notes">
						<ul>
							<li>Visually, this is what the process looks like in 2 dimensions</li>
							<li>Note: by replicating this process multiple times, we can be more confident in results if parallel replicates converge to the same thing</li>
						</ul>
					</aside>
				</section>
				<!-- BA.1 Pakistan -->
				<section>
					<h2>BA.1 in Pakistan</h2>
					<h4>A recent epidemic withing a global pandemic</h4>
					<aside class="notes">tmp</aside>
				</section>
				<section>
					<h3>Driving questions</h3>
					<ol>
						<li>How does a pathogen initially enter a country and influence local epidemics?</li>
						<li>How can we use phylodynamic methods to overcome global disparities in sequencing?</li>
					</ol>
					<aside class="notes">tmp</aside>
				</section>
				<section>
					<section>
						<h3>Data in this analysis</h3>
						<div id="left" style="font-size:80%">
							<ul>
								<li>1690 global BA.1 genomes</li>
								<ul>
									<li>63 from Pakistan</li>
									<li>15 with travel-histories</li>
								</ul>
								<li>32 discrete geographic areas</li>
								<ul>
									<li>Pakistan and neighboring countries</li>
									<li>Regions of China</li>
									<li>UN Georegions</li>
								</ul>
								<li>Roughly one third of Pakistan's fifth wave</li>
							</ul>
						</div>
						<div id="right">
							<img class="r-stretch" src="assets/doctoral-defense/pie_pk.png" width="100%" alt="" />
						</div>
					</section>
					<section>
						<img class="r-stretch" src="assets/2024-02-12-grubaugh-lab-meeting/pie.png" width="100%" alt="" />
					</section>
				</section>
				<section>
					<h3>Pakistan's Omicron epidemic</h3>
					<div id="left" style="font-size:80%">
						<img src="assets/doctoral-defense/pk_cases.png" alt="alt" width="100%" class="r-stretch">
					</div>
					<div id="right" style="font-size:80%">
						<img src="assets/doctoral-defense/pk_sequences.png" alt="alt" width="100%" class="r-stretch">
					</div>
					<aside class="notes">Quick overview of the Omicron epidemic in Pakistan in late '21/early '22</aside>
				</section>
				<section data-transition="slide-in fade-out">
					<img src="assets/doctoral-defense/pk_main1.png" alt="alt" width="100%" class="r-stretch">
					<aside class="notes">tmp</aside>
				</section>
				<section data-transition="fade">
					<img src="assets/doctoral-defense/pk_main2.png" alt="alt" width="100%" class="r-stretch">
					<aside class="notes">tmp</aside>
				</section>
				<section data-transition="fade">
					<img src="assets/doctoral-defense/pk_main3.png" alt="alt" width="100%" class="r-stretch">
					<aside class="notes">tmp</aside>
				</section>
				<section data-transition="fade-in slide-out">
					<img src="assets/doctoral-defense/pk_main4.png" alt="alt" width="100%" class="r-stretch">
					<aside class="notes">tmp</aside>
				</section>
				<section>
					<img src="assets/doctoral-defense/pk_map.png" alt="alt" width="100%" class="r-stretch">
					<aside class="notes">tmp</aside>
				</section>
				<section>
					<img src="assets/doctoral-defense/pk_sankey.png" alt="alt" width="100%" class="r-stretch">
					<aside class="notes">tmp</aside>
				</section>
				<section>
					<h3>Key Takeaways</h3>
					<ul>
						<li>BA.1 was the primary driver of the beginning of Pakistan's Omicron epidemic.</li>
						<li>Air traffic from Northern Europe caused most importations.</li>
						<li>Disparities in sequencing made analysis difficult and potentially biased.</li>
					</ul>
					<aside class="notes">
						just read this slide
					</aside>
				</section>
				<!-- HBV SPECIFIC STUFF -->
				<section>
					<h2>Phylogeography of Hepatitis B Virus</h2>
					<h4>A virus that has co-evolved with humans for millennia</h4>
				</section>
				<section>
					<h3>This study</h3>
					<ul>
						<li>133 HBV positive individuals who have recently immigrated from sub-Saharan Africa</li>
						<li>118 full genome sequences:</li>
						<ul>
							<li>47 HBV-A;</li>
							<li><em>7 HBV-D;</em></li>
							<li><em>64 HBV-E</em>.</li>
						</ul>
					<li>Supplemented with all available high quality GenBank HBV genomes</li>
					</ul>
					<aside class="notes">
						<li>2005-2010</li>
						<li>583 HBV-A sequences (dated 1979–2014)</li>
						<li>764 HBV-D sequences (dated 1975–2012)</li>
						<li>234 HBV-E sequences (dated 1994–2010)</li>
					</aside>
				</section>
				<section>
					<img class="r-stretch" src="assets/hbv-vilnius-university/hbv_genome_schematic.png" alt="hbv genome" />
					<div id="leftSource">Source: Wikimedia Commons</div>
					<aside class="notes">
						<ul>
							<li>Orthohepadnavirus</li>
							<li>reverse-transcribing partially dsDNA genome</li>
							<li>circular (-) sense strand, partial ()+ sense strand</li>
							<li>overlapping reading frames</li>
							<li>highly compact genome, with little room for mutation</li>
						</ul>
					</aside>
				</section>
				<section>
					<img src="assets/hbv-vilnius-university/full-hbv-tree.jpg" alt="alt" class="r-stretch" width="100%">
					<aside class="notes">tmp</aside>
					<div id="leftSource">Source: Kocher et al. (2021)</div>
				</section>
				<section>
					<h4>Estimating evolutionary rate from trees</h4>
					<img src="assets/doctoral-defense/tempSignal.png" alt="alt" width="100%" class="r-stretch">
					<aside class="notes">tmp</aside>
				</section>
				<section>
					<img class="r-stretch" src="assets/hbv-vilnius-university/muhlemann.png" alt="hbv sampling map" />
					<div id="leftSource">Source: Mülemann et al. (2018)</div>
					<aside class="notes">
						<li>Issue: clock rate of HBV is very hard to derive from empirical data because of its peculiar genome</li>
						<li>We seek to remedy by drawing on recent work on ancient HBV genomics to get more data and informative priors</li>
					</aside>
				</section>
				<section>
					\[\scriptsize
					\begin{array}{cccc}
					\textbf{Sequence} & \textbf{Genotype} & \textbf{Location} & \textbf{Age~(year)} \\ \hline
					\textrm{Rise386} & \textrm{HBV-A} & \textrm{Russia}^* & 4,114 \textrm{(2100 BCE)} \\
					\textrm{Rise387} & \textrm{HBV-A} & \textrm{Russia}^* & 4,278 \textrm{(2264 BCE)} \\
					\textrm{DA119} & \textrm{HBV-A} & \textrm{Slovakia} & 1,563 \textrm{(451 CE)} \\
					\textrm{DA195} & \textrm{HBV-A} & \textrm{Hungary} & 2,641 \textrm{(627 BCE)} \\
					\end{array}
					\]

					<p style="font-size:50%">Clock rate prior: $1.18 \times 10^{-5}$ [95% HPD: $8.04 \times 10^{-6} \textrm{--} 1.51 \times 10^{-5}$] subs./site/year</p>

					<div id="leftSource">Source: Mülemann et al. (2018)</div>

					<aside class="notes">
						<ul>
							<li>searched literature for high quality full ancient genomes</li>
							<li>4 HBV-A, 5 HBV-D</li>
							<li>None for HBV-E</li>
							<li>*Russian sequences are given a Central Asia location instead of Europe; proximal to Kazakhstan</li>
							<li>583 + 4 ancient</li>
						</ul>
					</aside>
				</section>
				<section>
					<h3>Driving questions</h3>
					<ol>
						<li>How do human movement patterns shape viral spread?</li>
						<li>Do ancient genomes help us to infer the timing of events in the distant past?</li>
					</ol>
					<aside class="notes">tmp</aside>
				</section>
				<section>
					<img class="r-stretch" src="assets/2024-02-12-grubaugh-lab-meeting/hbv/Figure4.png" width="100%">
					<aside class="notes">
						<ul>
							<li>Time calibrated MCC tree colored by location</li>
							<li>Broken axis, swoops on map</li>
							<li>2500 BCE tMRCA, introduction to Europe shortly after</li>
							<li>Potentially drivers of mid-millenium diversity?</li>
							<ul>
								<li>5 jumps Africa to Americas after 1700</li>
								<li>1520's-1850s</li>
							</ul>
							<li>Globalization after the 1960s</li>
						</ul>
					</aside>
				</section>
				<section>
					<h4>Migration counts and support</h4>
					<div id="left">
						<img class="r-stretch" src="assets/hbv-vilnius-university/mj_hbva.png" width="100%" alt="hbv a markov jumps" />
					</div>
					<div id="right">
						<img class="r-stretch" src="assets/hbv-vilnius-university/bf_hbva.png" width="100%" alt="hbv a bayes factors" />
					</div>
					<aside class="notes">
						<ul>
							<li>Europe drives exports</li>
							<li>Mainly to Americas and East Asia</li>
							<li>>11 Markov jumps each (with BF support)</li>
						</ul>
					</aside>
				</section>
				<!-- HBV-D -->
				<section>
					<h3>Takeaways and next steps</h3>
					<ul>
						<li>Large-scale human movements drives HBV globalization.</li>
						<li>More global genomic surveillance of HBV is necessary to understand the diversity and potential impact of each genotype.</li>
						<li>Find more mummies.</li>
					</ul>
					<aside class="notes">
						just read this slide
					</aside>
				</section>
				<section>
					<h3>Overall conclusions</h3>
					<ul>
						<li>(Bayesian) phylodynamics can help us understand both recent and ancient patterns of viral dispersal.</li>
						<li>Disparities in sequencing complicate the analysis of epidemics.</li>
						<li>We need more efficient ways to analyze increasingly large datasets.</li>
					</ul>
					<aside class="notes"></aside>
				</section>
				<!-- Acknowledgements slide -->
				<section data-background="#ffffff">
					<div class="row">
					<div class="column">
						<h4>ECV-KU Leuven</h4>
						<ul style="color:#282828">
							<li>Guy Baele</li>
							<li>Mandev Gill</li>
							<li>Philippe Lemey</li>
							<li>Samuel Hong</li>
						</ul>
						<p></p>
						<h4>Fogarty International Center</h4>
						<ul style="color:#282828">
							<li>Nídia Sequeira Trovão</li>
						</ul>
						<p></p>
						<h4>Vilnius University</h4>
						<ul style="color:#282828">
							<li>Gytis Dudas</li>
						</ul>
					</div>
					<div class="column">
						<img class="r-stretch" src="assets/share/kul_logo.png" alt="KU Leuven logo" width="60%"/>
						<img class="r-stretch" src="assets/share/ecv_logo.png" alt="ecv logo" width="40%"/>
						<img class="r-stretch" src="assets/share/fogarty_logo.png" alt="Fogarty logo" width="60%"/>
						<img class="r-stretch" src="assets/share/vilnius_university_logo.png" alt="nih pk logo" width="70%"/>
						<img class="r-stretch" src="assets/share/nih_pk.png" alt="baef logo" width="60%"/>
					</div>
					<div class="column">
						<h4>Rega Institute</h4>
						<ul style="color:#282828">
							<li>Mahmoud Reza Pourkarim</li>
							<li>Marijn Thijssen</li>
						</ul>
						<h4>NIH Pakistan</h4>
						<ul style="color:#282828">
							<li>Massab Umair</li>
							<li>Zaira Rehman</li>
							<li>Aamer Ikram</li>
							<li>Muhammad Salman</li>
						</ul>
						<p></p>
						<h4>Belgian American Education Foundation</h4>
					</div>
				</section>
				<section>
					<div id="left">
						<img src="assets/doctoral-defense/old_tree.png" class="r-stretch" width="80%">
					</div>
					<div id="right">
						<h3><br/><br/><br/><br/>Questions?</h3>
					</div>
				</section>
				<section>
					<h2>HBV-D</h2>
					<aside class="notes">
						764 + 5 ancient
					</aside>
				</section>
				<section>
					<div class="">

					</div>
					<img class="r-stretch" src="assets/2024-02-12-grubaugh-lab-meeting/hbv/Figure5.png" width="100%">
					<aside class="notes">
						<ul>
							<li>1800 BCE tMRCA</li>
							<li>92 total jumps</li>
							<ul>
								<li>more distributed throughout the evol histoyr</li>
								<li>matches: no geographic isolation allows continuous movement</li>
							</ul>
							<li>Large introduction to Africa ca. 550 CE</li>
							<li>Mixed European/American clade w/ tMRCA 1100 CE</li>
						</ul>
					</aside>
				</section>
				<section>
					<div id="left">
						<img class="r-stretch" src="assets/hbv-vilnius-university/mj_hbvd.png" width="100%" alt="hbv d markov jumps" />
					</div>
					<div id="right">
						<img class="r-stretch" src="assets/hbv-vilnius-university/bf_hbvd.png" width="100%" alt="hbv d bayes factors" />
					</div>
					<aside class="notes">
						<ul>
							<li>Central Asia, and to a smaller degree Europe drove spread</li>
							<li>makes sense given its known distribution</li>
						</ul>
					</aside>
				</section>
			</div>
		</div>

		<script src="dist/reveal.js"></script>
		<script src="plugin/notes/notes.js"></script>
		<script src="plugin/markdown/markdown.js"></script>
		<script src="plugin/highlight/highlight.js"></script>
		<script src="plugin/math/math.js"></script>
		<script>
			// More info about initialization & config:
			// - https://revealjs.com/initialization/
			// - https://revealjs.com/config/
		 Reveal.initialize({
			    controls: true,
				hash: true,
				slideNumber: true,
				hashOneBasedIndex: true,

				// Learn about plugins: https://revealjs.com/plugins/
				plugins: [ RevealMarkdown, RevealHighlight, RevealNotes, RevealMath.KaTeX, ]
			});
		</script>
	</body>
</html>
