<!doctype html>
<html>
	<head>
		<meta charset="utf-8">
		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">

		<title>HBV Migration</title>
		<meta name="author" content="Barney Isaksen Potter">
		<meta name="description" content="Human migration patterns shape hepatitis B diversity" />

		<link rel="stylesheet" href="dist/reset.css">
		<link rel="stylesheet" href="dist/reveal.css">
		<link rel="stylesheet" href="dist/theme/white.css">
		<link rel="stylesheet" href="css/custom.css">
		<!-- Theme used for syntax highlighted code -->
		<link rel="stylesheet" href="plugin/highlight/monokai.css">
	</head>
	<body>
		<div class="reveal">
			<div class="slides">
				<!-- BEGIN SLIDES -->
				<!-- Title slide -->
				<section data-background-color="#ebdbb2" data-background-image="assets/share/sedes_t.png"
						 data-background-size="35%"
						 data-background-position="125% 125%"
						 data-background-repeat="no-repeat"
						 data-background-opacity="0.5">
					<h2 style="color:#076678">Human migration patterns shape hepatitis B diversity</h2>
					<h4 style="color:#076678">Barney Isaksen Potter</h4>
					<p style="font-size:0.6em;color:#282828">KU Leuven</p>
					<p style="font-size:0.6em;color:#282828">2022-12-21</p>
					<p style="font-size:0.6em;color:#282828">Life Sciences Center, Vilnius University</p>
					<aside class="notes">
						<ul>
							<li>Thanks for being here!</li>
						</ul>
					</aside>
				</section>
				<section>
					<div class="r-stack">
						<img class="fragment current-visible" src="assets/share/phylogeo_intro/gruvbox_dark/epi1.png">
						<img class="fragment current-visible" src="assets/share/phylogeo_intro/gruvbox_dark/epi2.png">
						<img class="fragment current-visible" src="assets/share/phylogeo_intro/gruvbox_dark/epi3.png">
						<img class="fragment current-visible" src="assets/share/phylogeo_intro/gruvbox_dark/epi4.png">
						<img class="fragment current-visible" src="assets/share/phylogeo_intro/gruvbox_dark/epi5.png">
						<img class="fragment current-visible" src="assets/share/phylogeo_intro/gruvbox_dark/epi6.png">
						<img class="fragment current-visible" src="assets/share/phylogeo_intro/gruvbox_dark/epi7.png">
						<img class="fragment" src="assets/share/phylogeo_intro/gruvbox_dark/epi8.png">
					</div>
					<aside class="notes">
						<ul>
							<li>walk through figure</li>
							<li>phylogeny represents a hypothesis of the shared evolutionary, and therefore transmission, history</li>
						</ul>
					</aside>
				</section>
				<!-- Phylogeny explanation slide -->
				<section data-transition="fade">
					<h3>What can phylogenetics tell us?</h3>
					<div id="left">
						<ul>
							<li>Phylogeny</li>
							<li style="opacity:.5">Evolutionary rate</li>
							<li style="opacity:.5">Substitution rates</li>
							<li style="opacity:.5">Demographic parameters</li>
							<li style="opacity:.5">Migration rates</li>
							<li style="opacity:.5">Migration history</li>
						</ul>
					</div>
					<div id="right">
						<img class="r-stretch" src="assets/share/what_can_phylogeo/gb_dark/phylogeny.png" width="100%">
					</div>
					<aside class="notes">
						<ul>
							<li>Which things are most related to one another</li>
							<li>How close or distant different taxa are</li>
						</ul>
					</aside>
				</section>
				<!-- Evolutionary rate explanation slide -->
				<section data-transition="fade">
					<h3>What can phylogenetics tell us?</h3>
					<div id="left">
						<ul>
							<li style="opacity:.5">Phylogeny</li>
							<li>Evolutionary rate</li>
							<li style="opacity:.5">Substitution rates</li>
							<li style="opacity:.5">Demographic parameters</li>
							<li style="opacity:.5">Migration rates</li>
							<li style="opacity:.5">Migration history</li>
						</ul>
					</div>
					<div id="right">
						<img class="r-stretch" src="assets/share/what_can_phylogeo/gb_dark/time_scale.png" width="100%">
					</div>
					<aside class="notes">
						<ul>
							<li>Leverage clock-like nature of mutation accumulation to date internal nodes</li>
						</ul>
					</aside>
				</section>
				<!-- Substitution rate explanation slide -->
				<section data-transition="fade">
					<h3>What can phylogenetics tell us?</h3>
					<div id="left">
						<ul>
							<li style="opacity:.5">Phylogeny</li>
							<li style="opacity:.5">Evolutionary rate</li>
							<li>Substitution rates</li>
							<li style="opacity:.5">Demographic parameters</li>
							<li style="opacity:.5">Migration rates</li>
							<li style="opacity:.5">Migration history</li>
						</ul>
					</div>
					<div id="right">
						\[\tiny
						Q_{JC69} =
						\begin{bmatrix}
						\cdot & \lambda & \lambda & \lambda \\
						\lambda & \cdot & \lambda & \lambda \\
						\lambda & \lambda & \cdot & \lambda \\
						\lambda & \lambda & \lambda & \cdot \\
						\end{bmatrix} \\[1em]
						Q_{GTR} =
						\begin{bmatrix}
						\cdot & a\pi_C & b\pi_A & c\pi_G \\
						a\pi_T & \cdot & d\pi_A & e\pi_G \\
						b\pi_T & d\pi_C & \cdot & f\pi_G \\
						c\pi_T & e\pi_C & f\pi_A & \cdot \\
						\end{bmatrix}
						\]
					</div>
					<aside class="notes">
						<ul>
							<li>Nucleotide substitution rates</li>
							<li>Variety of models</li>
						</ul>
					</aside>
				</section>
				<!-- Demographic parameters explanation slide -->
				<section data-transition="fade">
					<h3>What can phylogenetics tell us?</h3>
					<div id="left">
						<ul>
							<li style="opacity:.5">Phylogeny</li>
							<li style="opacity:.5">Evolutionary rate</li>
							<li style="opacity:.5">Substitution rates</li>
							<li>Demographic parameters</li>
							<li style="opacity:.5">Migration rates</li>
							<li style="opacity:.5">Migration history</li>
						</ul>
					</div>
					<div id="right">
						<img class="r-stretch" src="assets/share/what_can_phylogeo/gb_dark/demographic_history.png" width="100%">
					</div>
					<aside class="notes">
						<ul>
							<li>Not just evol. characteristics, but also epidemiological ones</li>
							<li>Use the phylogeny to infer effective population size through time</li>
							<li>proxy for reproductive rate</li>
						</ul>
					</aside>
				</section>
				<!-- Migration rate explanation slide -->
				<section data-transition="fade">
					<h3>What can phylogenetics tell us?</h3>
					<div id="left">
						<ul>
							<li style="opacity:.5">Phylogeny</li>
							<li style="opacity:.5">Evolutionary rate</li>
							<li style="opacity:.5">Substitution rates</li>
							<li style="opacity:.5">Demographic parameters</li>
							<li>Migration rates</li>
							<li style="opacity:.5">Migration history</li>
					</div>
					<div id="right">
						<img class="r-stretch" src="assets/share/what_can_phylogeo/gb_dark/migration_rates.png" width="100%">
					</div>
					<aside class="notes">
						<ul>
							<li>Similar to substitution rates, we can infer migration rates</li>
							<li>Help us understand which locations may be sources or sinks for viral spread</li>
						</ul>
					</aside>
				</section>
				<!-- Migration rate history slide -->
				<section data-transition="fade">
					<h3>What can phylogenetics tell us?</h3>
					<div id="left">
						<ul>
							<li style="opacity:.5">Phylogeny</li>
							<li style="opacity:.5">Evolutionary rate</li>
							<li style="opacity:.5">Substitution rates</li>
							<li style="opacity:.5">Demographic parameters</li>
							<li style="opacity:.5">Migration rates</li>
							<li>Migration history</li>
						</ul>
					</div>
					<div id="right">
						<img class="r-stretch" src="assets/share/what_can_phylogeo/gb_dark/migration_history.png" width="100%">
					</div>
					<aside class="notes">
						<ul>
							<li>Combine migration rates with a phylogenetic tree</li>
							<li>Understand the number and timing of different migration events</li>
						</ul>
					</aside>
				</section>
				<!-- METHODS -->
                <section>
                    <blockquote>
                        How can we infer phylogenies robustly from genomic sequence data and geographic metadata?
                    </blockquote>
                </section>
				<!-- Tree space size -->
				<section>
					<h3>Problem: parameter space size</h3>
					\[\tiny
					\begin{array}{cc}
					Num.~taxa & Num.~topologies \\ \hline
					1 & 1 \\
					2 & 1 \\
					3 & 3 \\
					4 & 15 \\
					5 & 105 \\
					6 & 945 \\
					7 & 10,395 \\
					8 & 135,135 \\
					9 & 2,027,025 \\
					\vdots & \vdots \\
					769 & 3.753 \times 10^{2,110}
					\end{array}
					\]
				</section>
				<section>
					<img class="r-stretch" src="assets/share/phylogeo_intro/stamatakis2020_tree_space.png" width="100%">

					<div id="leftSource">Source: Stamatakis & Kozlov (2020)</div>
				</section>
				<section data-auto-animate>
					\[
					P(\theta|\textbf{D}) = \frac{P(\textbf{D} |\theta) \times P(\theta )}{P(\textbf{D})}
					\]
				</section>
				<section data-auto-animate>
					\[
					P(\theta|\textbf{grumpy}) = \frac{P(\textbf{D} |\theta) \times P(\theta )}{P(\textbf{D})}
					\]
				</section>
				<section>
					<img class="r-stretch" src="assets/share/phylogeo_intro/beast_likelihood.png" alt="likelihood calculation" width="100%"/>
				</section>
				<section>
					<img class="r-stretch" src="assets/share/phylogeo_intro/likelihood_surface.png" alt="likelihood calculation" width="50%"/>
				</section>
				<section data-background="https://revolution-computing.typepad.com/.a/6a010534b1db25970b019aff4a7bbc970d-pi" data-background-size="30em">
				</section>
				<section>
					<img class="r-stretch" src="assets/share/phylogeo_intro/beast_main.png" alt="likelihood calculation" width="70%"/>
				</section>
				<!-- HBV SPECIFIC STUFF -->
                <!-- HBV image -->
				<section>
					<img class="r-stretch" src="assets/hbv-vilnius-university/hbv-micrograph.jpg" alt="HBV micrograph" />
						<div id="leftSource">Source: CDC/ Dr. Erskine Palmer</div>
						<aside class="notes">
						<ul>
							<li>Main antagonist: HBV</li>
                            <li>list some facts</li>
						</ul>
					</aside>
				</section>
				<section>
					<h3>What is HBV?</h3>
					<div id="left" style="font-size:82%">
						<ul>
							<li>Causative agent of heptatitis B</li>
							<ul>
								<li>Chrrhosis</li>
								<li>Hepatocellular carcinoma</li>
							</ul>
							<li>Spread through blood and bodily fluids</li>
							<li>Affects $\geq 350,000,000$ people annually</li>
							<li>Partialy dsDNA genome</li>
							<ul>
								<li>Long strand: 3 kb</li>
								<li>Short strand: 2.2 kb</li>
							</ul>
						</ul>
					</div>
					<div id="right">
						<img class="r-stretch" src="assets/hbv-vilnius-university/hbv_genome_schematic.png" width="100%" alt="HBV genome" />
					</div>
					<aside class="notes">
						<ul>
							<li>partially dsDNA genome</li>
							<li>partially ciruclar</li>
							<li>overlapping reading frames</li>
						</ul>
					</aside>
				</section>
				<!-- EID slide -->
				<section>
					<img class="r-stretch" src="assets/second-year-seminar/emerging-infectious-diseases-morens-fauci-2020.jpg" alt="EIDs" />
						<div id="leftSource">Source: Morens & Fauci (2020)</div>
						<aside class="notes">
						<ul>
							<li>Increasingly interconnected, population dense world leads to more EIDs</li>
							<li>Range from isolated outbreaks (MRSA, MERS) to epidemics of regional scale (West African Ebola, Zika in the Americias) to pandemics of global scale (SARS-CoV-2)</li>
							<li>As EIDs continue to increasingly affect global public health, we need to develop tools to characterize them and inform public health interventions in a timely manner</li>
						</ul>
					</aside>
				</section>
                <!-- HBV tree -->
				<section>
					<img class="r-stretch" src="assets/hbv-vilnius-university/full-hbv-tree.jpg" alt="full hbv tree" />
					<div id="leftSource">Source: Kocher et al. (2021), <em>Science</em>.</div>
						<aside class="notes">
						<ul>
							<li>Main antagonist: HBV</li>
              <li>list some facts</li>
						</ul>
					</aside>
				</section>
				<section>
					HBV ethnospatial distribution slide
				</section>
				<section>
					Reconstruct evolutionary and migration history of three HBV genotypes (A, D, E):
					<ul>
						<li>Identify subgenotypes for novel sequences (A/D);</li>
						<li>Date the most recent common ancestor of each genotype;</li>
						<li>Characterize number and timing of major viral migrations.</li>
					</ul>
				</section>
				<section>
					<h4>Sampling bias complicates phyogeographic inference</h4>
					<img class="r-stretch" src="assets/hbv-vilnius-university/static_map_with_counts.png" alt="hbv sampling map" />
				</section>
				<section>
					<img class="r-stretch" src="assets/hbv-vilnius-university/muhlemann.png" alt="hbv sampling map" />
					<div id="leftSource">Source: Mülemann et al. (2018)</div>
				</section>
				<section>
					\[\scriptsize
					\begin{array}{cccc}
					\textbf{Sequence} & \textbf{Genotype} & \textbf{Location} & \textbf{Age~(year)} \\ \hline
					\textrm{Rise386} & \textrm{HBV-A} & \textrm{Russia}^* & 4,114 \textrm{(2100 BCE)} \\
					\textrm{Rise387} & \textrm{HBV-A} & \textrm{Russia}^* & 4,278 \textrm{(2264 BCE)} \\
					\textrm{DA119} & \textrm{HBV-A} & \textrm{Slovakia} & 1,563 \textrm{(451 CE)} \\
					\textrm{DA195} & \textrm{HBV-A} & \textrm{Hungary} & 2,641 \textrm{(627 BCE)} \\ \hline
					\textrm{DA27} & \textrm{HBV-D} & \textrm{Kazakhstan} & 1,610 \textrm{(409 CE)} \\
					\textrm{DA29} & \textrm{HBV-D} & \textrm{Kazakhstan} & 822 \textrm{(1197 CE)} \\
					\textrm{DA51} & \textrm{HBV-D} & \textrm{Kyrgyzstan} & 2,297 \textrm{(278 BCE)} \\
					\textrm{DA222} & \textrm{HBV-D} & \textrm{Kazakhstan} & 1,167 \textrm{(852 CE)} \\
					\textrm{NASD24SEQ} & \textrm{HBV-D} & \textrm{Italy} & 427 \textrm{(1587 CE)}
					\end{array}
					\]

					<p style="font-size:50%">Clock rate prior: $1.18 \times 10^{-5}$ [95% HPD: $8.04 \times 10^{-6} \textrm{--} 1.51 \times 10^{-5}$] subs./site/year</p>

					<div id="leftSource">Source: Mülemann et al. (2018), Ross et al. (2018)</div>
				</section>
				<!-- HBV-A -->
				<section>
					<h2>HBV-A</h2>
				</section>
				<section>
					<h4>ML sequence placement</h4>
					<img class="r-stretch" src="assets/hbv-vilnius-university/HBV-A_new_sequences.png" width="100%">
					<aside class="notes">
						<ul>
							<li>Combine migration rates with a phylogenetic tree</li>
							<li>Understand the number and timing of different migration events</li>
						</ul>
					</aside>
				</section>
				<section>
					<h4>Bayesian phylogeography</h4>
					<img class="r-stretch" src="assets/hbv-vilnius-university/HBV-A_phylogeography_and_mcc_tree.png" width="100%">
					<aside class="notes">
						<ul>
							<li>Combine migration rates with a phylogenetic tree</li>
							<li>Understand the number and timing of different migration events</li>
						</ul>
					</aside>
				</section>
				<section>
					<h4>Markov jump analysis</h4>
					<div id="left">
						<img class="r-stretch" src="assets/hbv-vilnius-university/mj_hbva.png" width="100%" alt="hbv a markov jumps" />
					</div>
					<div id="right">
						<img class="r-stretch" src="assets/hbv-vilnius-university/bf_hbva.png" width="100%" alt="hbv a bayes factors" />
					</div>
				</section>
				<!-- HBV-D -->
				<section>
					<h2>HBV-D</h2>
				</section>
				<section>
					<h3>HBV-D new sequences</h3>
					<img class="r-stretch" src="assets/hbv-vilnius-university/HBV-D_new_sequences.png" width="100%">
					<aside class="notes">
						<ul>
							<li>Combine migration rates with a phylogenetic tree</li>
							<li>Understand the number and timing of different migration events</li>
						</ul>
					</aside>
				</section>

				<section>
					<h3>HBV-D evolutionary history</h3>
					<img class="r-stretch" src="assets/hbv-vilnius-university/HBV-D_phylogeography_and_mcc_tree.png" width="100%">
					<aside class="notes">
						<ul>
							<li>Combine migration rates with a phylogenetic tree</li>
							<li>Understand the number and timing of different migration events</li>
						</ul>
					</aside>
				</section>
				<section>
					<h4>Markov jump analysis</h4>
					<div id="left">
						<img class="r-stretch" src="assets/hbv-vilnius-university/mj_hbvd.png" width="100%" alt="hbv d markov jumps" />
					</div>
					<div id="right">
						<img class="r-stretch" src="assets/hbv-vilnius-university/bf_hbvd.png" width="100%" alt="hbv d bayes factors" />
					</div>
				</section>
				<!-- HBV-E -->
				<section>
					<h2>HBV-E</h2>
				</section>
				<section>
					<h3>HBV-E new sequences</h3>
					<img class="r-stretch" src="assets/hbv-vilnius-university/HBV-E_new_sequences.png" width="100%">
					<aside class="notes">
						<ul>
							<li>Combine migration rates with a phylogenetic tree</li>
							<li>Understand the number and timing of different migration events</li>
						</ul>
					</aside>
				</section>
				<section>
					<h3>HBV-E evolutionary history</h3>
					<img class="r-stretch" src="assets/hbv-vilnius-university/HBV-E_phylogeography_and_mcc_tree.png" width="100%">
					<aside class="notes">
						<ul>
							<li>Combine migration rates with a phylogenetic tree</li>
							<li>Understand the number and timing of different migration events</li>
						</ul>
					</aside>
				</section>
				<section>
					<h4>Markov jump analysis</h4>
					<div id="left">
						<img class="r-stretch" src="assets/hbv-vilnius-university/mj_hbve.png" width="100%" alt="hbv e markov jumps" />
					</div>
					<div id="right">
						<img class="r-stretch" src="assets/hbv-vilnius-university/bf_hbve.png" width="100%" alt="hbv e bayes factors" />
					</div>
				</section>
				<!-- Acknowledgements slide -->
				<section data-background="#ffffff">
					<div class="row">
					<div class="column">
						<h4>KU/UZ Leuven</h4>
						<ul style="color:#282828">
							<li>Guy Baele</li>
							<li>Piet Maes</li>
							<li>Simon Dellicour</li>
							<li>Lize Cuypers</li>
							<li>Emmanuel André</li>
							<li>Samuel Hong</li>
							<li>Nena Bollen</li>
						</ul>
						<p></p>
						<h4>GIGA Research Institute</h4>
						<ul style="color:#282828">
							<li>Keith Durkin</li>
						</ul>
					</div>
					<div class="column">
						<img class="r-stretch" src="assets/share/kul_logo.png" alt="KU Leuven logo" width="60%"/>
						<img class="r-stretch" src="assets/share/giga_logo.png" alt="KU Leuven logo" width="60%"/>
						<img class="r-stretch" src="assets/share/fred_hutch_logo.png" alt="KU Leuven logo" width="60%"/>
						<img class="r-stretch" src="assets/share/unibas_logo.jpeg" alt="KU Leuven logo" width="60%"/>
						<img class="r-stretch" src="assets/share/unibern_logo.png" alt="KU Leuven logo" width="50%"/>
					</div>
					<div class="column">
						<h4>Fred Hutch</h4>
						<ul style="color:#282828">
							<li>Trevor Bedford</li>
							<li>John Huddleston</li>
							<li>Tom Sibley</li>
						</ul>
						<p></p>
						<h4>University of Basel</h4>
						<ul style="color:#282828">
							<li>Richard Neher</li>
						</ul>
						<p></p>
						<h4>University of Bern</h4>
						<ul style="color:#282828">
							<li>Emma Hodcroft</li>
						</ul>
						<p></p>
						<h4>COVID-19 Genomics Belgium Consortium</h4>
					</div>
					<aside class="notes">
						<ul>
							<li>Thank Trevor and Richard</li>
							<li>covid 19 genomics belgium cons.</li>
						</ul>
					</aside>
				</section>
			</div>
		</div>

		<script src="dist/reveal.js"></script>
		<script src="plugin/notes/notes.js"></script>
		<script src="plugin/markdown/markdown.js"></script>
		<script src="plugin/highlight/highlight.js"></script>
		<script src="plugin/math/math.js"></script>
		<script>
			// More info about initialization & config:
			// - https://revealjs.com/initialization/
			// - https://revealjs.com/config/
		 Reveal.initialize({
			    controls: true,
				hash: true,
				slideNumber: true,
				hashOneBasedIndex: true,

				// Learn about plugins: https://revealjs.com/plugins/
				plugins: [ RevealMarkdown, RevealHighlight, RevealNotes, RevealMath.KaTeX, ]
			});
		</script>
	</body>
</html>
